knora/dsplib/utils/excel_to_json_project.py

```diff
@@ -0,0 +1,111 @@
+import json
+import os
+import re
+
+from knora.dsplib.models.helpers import BaseError
+from knora.dsplib.utils.excel_to_json_lists import excel2lists
+from knora.dsplib.utils.excel_to_json_properties import excel2properties
+from knora.dsplib.utils.excel_to_json_resources import excel2resources
+
+
+def excel2project(
+    data_model_files: str,
+    path_to_output_file: str
+) -> None:
+    """
+    Converts a folder containing Excel files into a JSON data model file. The folder must be structured like this:
+
+    ::
+
+        data_model_templates
+        |-- lists
+        |   |-- de.xlsx
+        |   `-- en.xlsx
+        `-- onto_name (onto_label)
+            |-- properties.xlsx
+            `-- resources.xlsx
```

Comment:
this kind of setup doesn't seem very user friendly to me... but I suppose this was identified as a requirement?

Reply:
Thanks for the hint. I'm going to ask some RDU people what they think.